src/Microsoft.AspNetCore.Server.IntegrationTesting/project.json

```diff
@@ -26,6 +26,7 @@
   },
   "dependencies": {
     "Microsoft.AspNetCore.Testing": "1.1.0-*",
+    "Microsoft.DotNet.PlatformAbstractions": "1.1.0-*",
```

Comment:
Can we use https://github.com/aspnet/Common/blob/dev/src/Microsoft.Extensions.RuntimeEnvironment.Sources/RuntimeEnvironment.cs instead?